source/fundamentals/connection/connection-options.txt

```diff
@@ -73,6 +73,28 @@ parameters of the connection URI to specify the behavior of the client.
      - Specifies whether to force dispatch **all** operations to the host
        specified in the connection URI.
 
+   * - **enableUtf8Validation**
+     - boolean
+     - ``true``
+     - 
+       Specifying ``true`` enables UTF-8 validation for the 
+       connection. MongoDB throws an error when 
+       it attempts to convert data that contains invalid 
+       UTF-8 characters. The validation adds validation processing 
+       overhead since it needs to check the data.
+
+       Specifying ``false`` disables UTF-8 validation for the 
+       connection. MongoDB does not throw errors when 
+       data contains invalid UTF-8 data. If you disable validation, 
+       your application avoids the validation processing overhead 
+       but cannot guarantee consistent presentation of invalid 
+       UTF-8 data. **Editing data** with ``enableUtf8Validation=false`` 
```

Comment:
Suggestion:

I think it's strange to format the full parameter and setting as it would appear in the connection string, especially since it's in the "enableUtf8Validation" parameter row and in the paragraph describing the "false" section.

```suggestion
       UTF-8 data. **Editing data** while validation is disabled
```